WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -33,124 +118,197 @@ class PersonViewController : UITableViewController {
 
         switch cell {
         case roleCell:
-            handleRoleWasPressed()
+            roleWasPressed()
         case removeCell:
-            handleRemoveWasPressed()
+            removeWasPressed()
         default:
             break
         }
     }
     
+    
+    
     // MARK: - Storyboard Methods
+    
     override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
         guard let roleViewController = segue.destinationViewController as? RoleViewController else {
             return
         }
         
         roleViewController.role = person.role
         roleViewController.onChange = { newRole in
-// TODO: JLP May.3.2016. To be implemented as part of #5175
+            self.updateRole(newRole)
         }
     }
     
     
-    // MARK: - Action Handlers
-    @IBAction func handleRoleWasPressed() {
+    
+    // MARK: - Constants
+    private let roleSegueIdentifier = "editRole"
+    private let gravatarPlaceholderImage = UIImage(named: "gravatar.png")
+}
+
+
+
+// MARK: - Private Helpers: Actions
+//
+private extension PersonViewController {
+    
+    func roleWasPressed() {
         performSegueWithIdentifier(roleSegueIdentifier, sender: nil)
     }
     
-    @IBAction func handleRemoveWasPressed() {
+    func removeWasPressed() {
 // TODO: JLP May.3.2016. To be implemented as part of #5175
```

Comment:
Is this todo out of date?  The PR description says its closing 5175.  Maybe it should be updated to 5288?